appveyor-nightly.yml

```diff
@@ -45,6 +45,7 @@ before_build:
 after_build:
   - rake "create_setup[%APPVEYOR_BUILD_VERSION%, %CONFIGURATION%, C:/smartxls.zip, 2.6.2.9]"
   - 7z a setup.zip %APPVEYOR_BUILD_FOLDER%\setup\deploy\*.msi
+  - rake "create_portable_setup[%APPVEYOR_BUILD_VERSION%, %CONFIGURATION%, pksim-portable-setup.zip]"
```

Comment:
if you mean artifact, then yes it is pushed on appveyor 
https://ci.appveyor.com/project/open-systems-pharmacology-ci/pk-sim/build/artifacts